backend/src/test/java/pl/zzpj/esportbetting/rest/UserRestControllerTest.java

```diff
@@ -104,7 +104,7 @@ public void whenFindByIdThenUserGiven() throws Exception {
                 .andExpect(jsonPath("$.email", Matchers.is(user.getEmail())))
                 .andExpect(jsonPath("$.firstName", Matchers.is(user.getFirstName())))
                 .andExpect(jsonPath("$.lastName", Matchers.is(user.getLastName())))
-                .andExpect(jsonPath("$.levelId",
+                .andExpect(jsonPath("$.level.id",
```

Comment:
Jako że w obiekcie mamy cały obiekt levelu, to chyba w testach też powinniśmy porównywać cały obiekt, a nie tylko jedną z jego wartości